bool Arm_isChecked
---
bool Arm_Abled
